src/Express.res

```diff
@@ -132,6 +133,9 @@ let is = (req, value) => req->is(value)->parseValue
 @send external jsonp: (res, 'a) => res = "jsonp"
 @send external links: (res, Js.Dict.t<string>) => res = "links"
 @send external location: (res, string) => res = "location"
+@send external on: (res, @string [
```

Comment:
I'd rather expose a `onClose` with a fixed `close` (`@as(json`"close"`)`), that'd simplify the overhead

Reply:
Totally agree, was just following official docs. Updated PR

```res
@send external onClose: (res, @as("close") _, unit => unit) => unit = "on"
```

```js
app.get("/stream", (function (req, res) {
        res.set("Cache-Control", "no-cache");
        res.set("Content-Type", "text/event-stream");
        res.set("Connection", "keep-alive");
        req.flushHeaders();
        res.on("close", (function (param) {
                console.log("Stream Ended");
                
              }));
        res.write("data: Hello World");
        res.write("data: Hello World");
        res.end();
        
      }));
      ```